ajax/libs/d3-cloud/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "d3-cloud",
+  "filename": "d3.layout.cloud.min.js",
+  "version": "1.2.4",
+  "description": "Generate word clouds in JavaScript.",
+  "keywords": [
+    "word",
+    "cloud",
+    "tag",
+    "visualization",
+    "canvas"
+  ],
+  "homepage": "https://www.jasondavies.com/wordcloud/",
```

Comment:
@pvnr0082t I reserved it because it's in the original `package.json`. Should I still remove it?